# Useful Links

A curated list of proteomics resources, tools, and documentation.

## Databases & Repositories

- **[PRIDE Archive](https://www.ebi.ac.uk/pride/)** - World's largest proteomics repository
- **[ProteomeXchange](https://www.proteomexchange.org/)** - Consortium of proteomics repositories
- **[MassIVE](https://massive.ucsd.edu/)** - UCSD proteomics repository
- **[jPOST](https://jpost.org/)** - Japan ProteOme STandard repository
- **[UniProt](https://www.uniprot.org/)** - Protein sequence and annotation database

## File Format Specifications

- **[mzTab Specification](https://github.com/HUPO-PSI/mzTab)** - Standardized MS results format
- **[mzML Format](http://www.psidev.info/mzml_1_0_0)** - Raw MS data format
- **[mzIdentML](http://www.psidev.info/mzidentml)** - Peptide/protein identification format

## Tools & Software

- **[MaxQuant](https://www.maxquant.org/)** - Popular proteomics software
- **[Proteome Discoverer](https://www.thermofisher.com/)** - Thermo Fisher software
- **[Perseus](https://maxquant.net/perseus/)** - Statistical analysis for proteomics
- **[Skyline](https://skyline.ms/)** - Targeted proteomics analysis

## Python Libraries

- **[Pandas](https://pandas.pydata.org/)** - Data manipulation
- **[NumPy](https://numpy.org/)** - Numerical computing
- **[SciPy](https://scipy.org/)** - Scientific computing
- **[Matplotlib](https://matplotlib.org/)** - Plotting
- **[Seaborn](https://seaborn.pydata.org/)** - Statistical visualization
- **[Plotly](https://plotly.com/python/)** - Interactive plots

## Workflow Management

- **[Snakemake](https://snakemake.readthedocs.io/)** - Workflow management system
- **[Nextflow](https://www.nextflow.io/)** - Data-driven workflows
- **[Docker](https://docs.docker.com/)** - Containerization

## Learning Resources

### Proteomics

- **[Nature Protocols: Proteomics](https://www.nature.com/nprot/)**
- **[Proteomics Academy](https://proteomics-academy.org/)**
- **[EBI Training](https://www.ebi.ac.uk/training/)** - Free proteomics courses

### Statistics

- **[Statistics for Proteomics](https://www.nature.com/articles/nmeth.3901)**
- **[Multiple Testing Correction](https://www.nature.com/articles/nbt1209-1135)**

### Python & Data Science

- **[Python Data Science Handbook](https://jakevdp.github.io/PythonDataScienceHandbook/)**
- **[Pandas Tutorial](https://pandas.pydata.org/docs/getting_started/intro_tutorials/index.html)**
- **[Real Python](https://realpython.com/)** - Python tutorials

### Bioinformatics

- **[Rosalind](http://rosalind.info/)** - Learn bioinformatics through problems
- **[Bioconductor](https://www.bioconductor.org/)** - R packages for bioinformatics
- **[Galaxy Project](https://galaxyproject.org/)** - Web-based bioinformatics platform

## Papers & Reviews

- **[ProteomeXchange 2023 Update](https://academic.oup.com/nar/article/51/D1/D1539/6825347)**
- **[PRIDE 2025 Update](https://academic.oup.com/nar/article/53/D1/D543/7484362)**
- **[Mass Spectrometry-based Proteomics](https://www.nature.com/articles/nature02046)**

## Communities & Forums

- **[Biostars](https://www.biostars.org/)** - Bioinformatics Q&A
- **[Stack Overflow](https://stackoverflow.com/)** - Programming Q&A
- **[Reddit r/proteomics](https://www.reddit.com/r/proteomics/)**
- **[ResearchGate](https://www.researchgate.net/)** - Scientific network

## YouTube Channels

- **[iBiology](https://www.youtube.com/c/iBiology)** - Scientific talks
- **[StatQuest](https://www.youtube.com/c/joshstarmer)** - Statistics explained simply
- **[Corey Schafer](https://www.youtube.com/c/Coreyms)** - Python tutorials

## Blogs & Websites

- **[The Molecular Ecologist](https://www.molecularecologist.com/)**
- **[Simply Statistics](https://simplystatistics.org/)**
- **[Towards Data Science](https://towardsdatascience.com/)**
